app/pb_kits/playbook/pb_text_input/_text_input.scss

```
@@ -8,51 +8,50 @@
 
   .text_input_wrapper {
     margin-bottom: 1rem;
-  }
-
-  .pb_text_input_kit {
-    border: 1px solid $border-light;
-    border-radius: $border-rad-light;
-    display: block;
-    width: 100%;
-    color: $text_lt_default;
-    background: $white;
-    padding: $space_xs $space_sm;
-    transition: 0.2s all;
-    outline: none;
-    font-size: $font_small;
-    margin: .1rem 0 0;
-
-    &:focus {
-      border-color: $primary-action;
-    }
-
-    &::-webkit-input-placeholder {
-      color: $text_lt_lighter;
-      opacity: 1;
-    }
-
-    &:-ms-input-placeholder {
-      color: $text_lt_lighter;
-      opacity: 1;
-    }
-
-    &::-ms-input-placeholder {
-      color: $text_lt_lighter;
-      opacity: 1;
-    }
-
-    &::placeholder {
-      color: $text_lt_lighter;
-      opacity: 1;
-    }
-
-    &:-ms-input-placeholder {
-      color: $text_lt_lighter;
-    }
-
-    &::-ms-input-placeholder {
-      color: $text_lt_lighter;
+    input {
```

I like everything else in this PR. This `input` is the one thing I put in the hands of UX to approve.

---

I get it since it’s a block kit. Is there anything we can do to make this more specific? Wish we could add a class to whatever input they pass

---

I could change it to this to it would match the first immediate child that is a input.

```suggestion
    & > input:first-child {
```

---

I like the update you made. I agree with the second as well. Might be something worth looking at in the future